src/HttpServer/Response.gren

```diff
@@ -0,0 +1,52 @@
+module HttpServer.Response exposing
+    ( Response(..)
```

Comment:
Should this be opaque?

Reply:
Yes. I'm struggling with the API though. Since `HttpServer` is the effect module it needed the Response variant to get the underlying response values to send to the Kernel. I could move the kernel calls to `Response.send` but then I'd have to expose it. I'm thinking I might have to merge `Response` back into `HttpRequest`.

Reply:
Or right, Response could have the cmd effects and httpserver could have the subs.